source/whats-new.txt

```diff
@@ -0,0 +1,60 @@
+.. _ruby-whats-new:
+
+==========
+What's New
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: update, new feature, deprecation, upgrade
+
+Learn what's new in:
+
+* :ref:`2.21 <version-2.21>`
+* :ref:`2.20 <version-2.20>`
+
+.. _upcoming-breaking-changes:
+
+.. _version-2.21:
+
+What's New in 2.21
+------------------
+
+The {+driver-short+} 2.21 release includes the following new features: 
+
+- Supports the Client-Side Operations Timeout (CSOT) feature, which unifies
+  most timeout-related options under a single ``timeout_ms`` option.
+- Supports {+mdb-server+} version 8.0.
+- Support for range v2 queries with Queryable Encryption, including a new ``trim_factor``
+  parameter. For more information about Queryable Encryption, see :manual:`Queryable Encryption </core/queryable-encryption>`
+  in the {+mdb-server+} manual.
+
```

Comment:
[question for technical reviewer] Does your team also want to link to driver release notes for each version at the end of the summary? Not all "What's New" pages do this but I noticed to the [current docs page](https://www.mongodb.com/docs/ruby-driver/current/release-notes/) links out so wanted to verify the preference of the team.

Reply:
See the [Node What's New docs](https://www.mongodb.com/docs/drivers/node/current/whats-new/#what-s-new-in-6.12) as an example of where the link would go.

Reply:
Linking to the specific versions might be most convenient for the user, if that's not too much trouble for the docs team.

@alexbevi might have some thoughts?

Reply:
It's no trouble 👍

Reply:
Updated the PR with the links in case @alexbevi wants to take a look and lmk if that looks good to go.

Reply:
I like the links to the GH release. LGTM